pom.xml

```diff
@@ -249,6 +249,12 @@
             <scope>runtime</scope>
         </dependency>
         <!--        Test dependencies -->
+        <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness</artifactId>
+            <version>1784.v83a_b_3ce99392</version>
```

Comment:
Without this explicit dependency, we use `1498.v53acb0fd4634` from a year ago which used jenkins-test-harness-htmlunit version 3 instead of 92, and version 3 was not compatible with [recent changes to the validation javascript](https://github.com/jenkinsci/jenkins/commit/848883db8d748316175930aa5fef3749ec5cdc46#diff-10922aee3260853be495a09cf3bf13bf5f931bc128e334807e3008ff00b57651R542) as this was only added in the sourceforge htmlunit library [this commit in December](https://github.com/HtmlUnit/htmlunit/commit/75e5f34587cfe3bb567799e981cf975574bc7cca)